### ISSUE TITLE
Low FPS in ALL games

### ISSUE THREAD

**Issue by GoogleCodeExporter** [GivePass/x360ce#103]
```
What steps will reproduce the problem?
1. Run x360ce
2. Run the game
3. Monitor the frame rate (FRAPS)

What is the expected output? What do you see instead?
Less than half the frame rate compared to running the game without x360ce

What version of x360ce and Windows are you using?
Windows 7 Professional x64, 2.0.2.90 (3.2.0.335 for the xinput1_3.dll)

What gamepad are you using? On what version of drivers?
Logitech Cordless Rumblepad 2

Basically, running x360ce will severely bring down my frame rate in ALL of my 
games. I was wondering why I kept getting such low frame rates in GTA IV! My 
FPS nearly triple when not using x360ce. I also noticed that x360ce uses around 
35% CPU usage when it's just sitting there doing nothing.
```

Original issue reported on code.google.com by `beswe...@gmail.com` on 16 Nov 2010 at 7:45
